example/example.client.ts

```diff
@@ -428,14 +428,34 @@ const substitute = (path: string, params: Record<string, any>) => {
   return [path, rest] as const;
 };
 
+const hasSameOrigin = (url: URL) => window.location.origin === url.origin;
+
 export type Implementation = (
   method: Method,
   path: string,
   params: Record<string, any>,
 ) => Promise<any>;
 
+const defaultImplementation: Implementation = async (method, path, params) => {
+  const hasBody = !["get", "delete"].includes(method);
+  const searchParams = hasBody ? "" : `?${new URLSearchParams(params)}`;
+  const url = new URL(`${path}${searchParams}`, "https://example.com");
+  const response = await fetch(url, {
+    mode: hasSameOrigin(url) ? "same-origin" : "cors",
```

Comment:
it turned out it's default to `cors`

https://developer.mozilla.org/en-US/docs/Web/API/RequestInit#mode
> Defaults to cors.

so, it's not needed

Reply:
reverted in 5e871cb6